ffi/capi/Cargo.toml

```diff
@@ -45,7 +45,6 @@ buffer_provider = [
     "icu_collator?/serde",
     "icu_datetime?/serde",
     "icu_decimal?/serde",
-    "icu_displaynames?/serde",
```

Comment:
I disabled displaynames FFI for now. We have to decide between pulling in all of experimental or not having it on FFI.

Reply:
I think it should still take part in DCE and I wouldn't want to forbid a component from having FFI only because it is in icu_experimental.

Also, we said that moving the code from icu_experimental into its own crate should be a no-op PR that puts the repo into a releasable state with the crate at stable, and we can't do that if we then need to add FFI bindings. We should start drafting the FFI bindings while the code is in icu_experimental.

I lean toward including it in `icu_capi` and probably also in `icu::experimental`

Reply:
We probably want to include it in `icu_capi`, but behind a `experimental_components` features (like in datagen), instead of one feature per module.

Regarding `icu`, I like the property that a module is either in `icu` or in `icu_experimental`, and that stabilisation moves it over. I also see value in removing the `experimental` feature from `icu`, we have too many features.

Reply:
Actually I retract what I said about `icu::experimental`. I think we should include it in `icu_capi`.